lua/lspconfig/configs.lua

```diff
@@ -257,16 +257,29 @@ function configs.__newindex(t, config_name, config_def)
       end
       local buf_path = util.path.sanitize(bufname)
 
-      if get_root_dir then
-        root_dir = get_root_dir(buf_path, bufnr)
+      local function check_fast(root_dir)
+        if not vim.in_fast_event() then
+          manager.add(root_dir, false, bufnr)
```

Comment:
Second argument can't be false all the time, it used to be true on the `config.single_file_support` branch